Build/libHttpClient.UnitTest.141.TAEF/libHttpClient.UnitTest.141.TAEF.vcxproj

```diff
@@ -56,11 +56,7 @@
   <ItemDefinitionGroup>
     <ClCompile>
       <AdditionalOptions>/bigobj /Zm512 /GS %(AdditionalOptions)</AdditionalOptions>
-      <AdditionalIncludeDirectories>
-       c:\Program Files (x86)\Windows Kits\10\Testing\Development\inc;
-       $(MSBuildThisFileDirectory)..\..\Source\Task;
-      %(AdditionalIncludeDirectories)
-      </AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>c:\Program Files (x86)\Windows Kits\10\Testing\Development\inc;$(MSBuildThisFileDirectory)..\..\Source\Task;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
Why one-line this, doesn't look like the contents changed?

Reply:
its just visual studio auto-formatting it